src/kernel/arch/x86/arch.zig

```diff
@@ -319,6 +329,14 @@ pub fn initMem(mb_info: BootPayload) std.mem.Allocator.Error!MemProfile {
     log.logInfo("Init mem\n", .{});
     defer log.logInfo("Done mem\n", .{});
 
+    log.logDebug("KERNEL_ADDR_OFFSET:    0x{X}\n", .{@ptrToInt(&KERNEL_ADDR_OFFSET)});
+    log.logDebug("KERNEL_STACK_START:    0x{X}\n", .{@ptrToInt(&KERNEL_STACK_START)});
+    log.logDebug("KERNEL_STACK_END:      0x{X}\n", .{@ptrToInt(&KERNEL_STACK_END)});
+    log.logDebug("KERNEL_VADDR_START:    0x{X}\n", .{@ptrToInt(&KERNEL_VADDR_START)});
+    log.logDebug("KERNEL_VADDR_END:      0x{X}\n", .{@ptrToInt(&KERNEL_VADDR_END)});
+    log.logDebug("KERNEL_PHYSADDR_START: 0x{X}\n", .{@ptrToInt(&KERNEL_PHYSADDR_START)});
+    log.logDebug("KERNEL_PHYSADDR_END:   0x{X}\n", .{@ptrToInt(&KERNEL_PHYSADDR_END)});
+
```

Comment:
Hopefully debugging this is over so let's purge these from existence

Reply:
I was thinking that this will be used to keep the exports in kmain up to date

Reply:
Keep them up to date?

Reply:
As we add code to the kernel, these values will change so can update the values in kmain that export these values for the unit testing so can keep things correct